### PR TITLE
CHANGELOG: add release notes for 0.1.0

### PR DESCRIPTION
**Pull request by LiliC**
This PR introduces a `CHANGELOG` file. Also added the notes that went to the `0.1.0` release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+## 0.1.0/2017-10-9
+
+First release of Habitat operator. Highlight of features:
+
+* [FEATURE] Support for creating/managing Habitat Services
+* [FEATURE] Support for different Habitat topologies
+* [FEATURE] Support for bind feature
+* [FEATURE] Support for initial configuration
+* [FEATURE] Support for encrypted ring communication
```
Comment (zeenix) [inline CHANGELOG.md]: I'm unable to appreciate redundancy unless it's absolutely required. I guess  "[FEATURE]" tag is a common practice so needs to be there but I don't see the point of "Support for " in the beginning of every feature title/description (it's implied).

Comment (LiliC) [reply CHANGELOG.md]: As discussed offline: We already have this in the Release notes, this is just importing it to the `CHANGELOG` file.

**Comment by krnowak** [comment]
An opinion: we should just close it. If we want to follow Habitat's change log, then you can notice that it does not have an entry for the first released version. Which makes sense, because there is no previous release on top of which the current release has improvements. So let's add the CHANGELOG when we actually release 0.2.0.

**Comment by LiliC** [comment]
@krnowak Agree with that. Closing. 👍

**Pull request closed without merging**